erpnext/accounts/report/cost_center_financial_statements/cost_center_financial_statements.js

```diff
@@ -10,7 +10,18 @@ frappe.query_reports["Cost Center Financial Statements"] = {
 			"fieldtype": "Link",
 			"options": "Company",
 			"default": frappe.defaults.get_user_default("Company"),
-			"reqd": 1
+			"reqd": 1,
+			"on_change": function(query_report) {
+				var company = query_report.get_filter_value("company");
+				if (!company) {
+					return;
+				}
+				frappe.model.with_doc("Company", company, function(r) {
```

Comment:
I guess you get the document in the callback, you wont have to do a get_doc, please check that

Reply:
frappe.model.with_doc returns the Company name in call back. get_doc is required to fetch the company cost center details